core/src/main/scala/org/apache/spark/deploy/DeployMessage.scala

```diff
@@ -64,7 +64,7 @@ private[deploy] object DeployMessages {
    * @param id the worker id
    * @param worker the worker endpoint ref
    */
-  case class WorkerDecommission(
```

Comment:
I think I messed up when reviewing this: I am so sorry :-(

Somehow I thought that `WorkerDecommission` is something being introduced in this PR, but apparently it is an existing message that went in #26440 itself.

Let's please keep the original message WorkerDecommission everywhere to avoid un-necessary merge conflicts etc in people's private forks :-)

At least I learned not to trust "changes since last reviewed" feature of Github when we all just force push into our PR branches :-)

Reply:
No stress, I did it as a seperate commit so I can revert it easily.